src/SDK/Client/Platform/methods/names/register.ts

```diff
@@ -35,25 +38,21 @@ export async function register(this: Platform,
     const [label] = nameLabels;
     const normalizedLabel = label.toLowerCase();
 
-    const preorderSalt = entropy.generate();
+    const preorderSalt = bs58.decode(entropy.generate());
```

Comment:
This won't work. Contract requires just random 32 bytes. Let's use `crypto.randomBytes(32)`;